eslint.yaml

```diff
@@ -32,7 +32,7 @@ rules:
   guard-for-in: 2
   no-else-return: 2
   no-loop-func: 2
-  vars-on-top: 2
+  vars-on-top: 1
```

Comment:
This looks it's being relaxed. Was that intentional?

@jeff-phillips-18 @dtaylor113

Reply:
Good catch, was unintentional but if we're good with it, which it sounds like we may be I can leave it be

Reply:
Sounds good to me!